src/components/Navigation.tsx

```diff
@@ -1,29 +1,36 @@
+"use client";
+
+import { usePathname } from "next/navigation";
+
 import { NavLink } from "@/components/button";
-import { Icon } from "@/components/icon/Icon";
+import { Icon, IconType } from "@/components/icon/Icon";
 
-const labels = [
-  { href: "/search", label: "話題を検索", icon: <Icon type="Search" /> },
-  { href: "/fav", label: "お気に入り", icon: <Icon type="Heart" /> },
-  { href: "/list", label: "買い物リスト", icon: <Icon type="ShoppingCart" /> },
+const LABELS = [
+  { href: "/search", label: "検索", icon: "Search" },
+  { href: "/fav", label: "お気に入り", icon: "Heart" },
+  { href: "/list", label: "買い物リスト", icon: "ShoppingCart" },
 ];
 
-// TODO: 仮の左カラム兼、フッターなので、後で対応お願いします。
-
-type navigationProps = {
-  pathname?: "/" | "/fav" | "list";
-};
+export const Navigation: React.FC = () => {
+  const pathname = usePathname();
 
-export const Navigation: React.FC<navigationProps> = (props) => {
   return (
     <nav className="pt-2">
       <ul className="flex w-screen justify-around sm:w-full sm:flex-col">
         <li className="hidden sm:block">ロゴ</li>
-        {labels.map(({ href, label, icon }) => {
+        {LABELS.map(({ href, label, icon }) => {
+          const isActive = pathname === href;
           return (
             <div key={href}>
-              <NavLink href={href} activeClassName="text-tomato" activeStrokeColor="tomato">
+              <NavLink
+                href={href}
+                activeClassName={isActive ? "text-tomato" : ""}
+                activeColor={isActive ? "tomato" : "black"}
+              >
                 <div className="mt-3 flex flex-row justify-start">
-                  <span className="">{icon}</span>
+                  <span className="">
+                    <Icon type={icon as IconType} size="large" color={isActive ? "tomato" : "black"} />
+                  </span>
```

Comment:
spanタグいらなそうなので削って良いかと。あと親にitems-centerあるとなお良しです。

```suggestion
                    <Icon type={icon} size="large" color={isActive ? "tomato" : "black"} />
```